charts/nexus3/files/configure.sh

```diff
@@ -149,40 +149,10 @@ do
     fi
   done
 
-  for json_file in "${base_dir}"/conf/*-role.json
-  do
-    if [ -f "${json_file}" ]
-    then
-      id="$(grep -Pio '(?<="id":)\s*\"[^"]+\"' "${json_file}" | xargs)"
-      source="$(grep -Pio '(?<="source":)\s*\"[^"]+\"' "${json_file}" | xargs)"
-      echo "Updating role '${id}'..."
-
-      status_code=$(curl -s -o /dev/null -w "%{http_code}" -X GET -H 'Content-Type: application/json' -u "${root_user}:${root_password}" "${nexus_host}/service/rest/beta/security/roles/${id}?source=${source}")
-      if [ "${status_code}" -eq 200 ]
-      then
-        status_code="$(curl -s -o /dev/null -w "%{http_code}" -X PUT -H 'Content-Type: application/json' -u "${root_user}:${root_password}" -d "@${json_file}" "${nexus_host}/service/rest/beta/security/roles/${id}")"
-        if [ "${status_code}" -ne 204 ]
-        then
-          echo "Could not configure role." >&2
-          exit 1
-        fi
-      else
-        status_code="$(curl -s -o /dev/null -w "%{http_code}" -X POST -H 'Content-Type: application/json' -u "${root_user}:${root_password}" -d "@${json_file}" "${nexus_host}/service/rest/beta/security/roles")"
-        if [ "${status_code}" -ne 200 ]
-        then
-          echo "Could not configure role." >&2
-          exit 1
-        fi
-      fi
-
-      echo "Role configured."
-    fi
-  done
-
   json_file="${base_dir}/conf/anonymous-user.json"
   if [ -f "${json_file}" ]
   then
-    echo "Configuring metrics..."
+    echo "Configuring anonymous user..."
```

Comment:
The anonymous user is an implementation detail of metrics which is why it wasn't previously referenced, I'd be happier if this was changed to `echo "Configuring anonymous user for metrics..."` (with the same logic for the completed message).

Reply:
Ok, will do.

Reply:
Updated.